platform/templates/kafka/kafka.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: streaming-service
+spec:
+  kafka:
+    replicas: 3
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+    storage:
+      {{ if .Values.kafka.storage.enable }}
+      type: persistent-claim
+      size: .Values.kafka.storage.storageSize
+      deleteClaim: false
+      {{ else }}
+      type: ephemeral
+      {{ end }}
+    {{ if .Values.kafka.storage.enable }}
```

Comment:
Why does having storage enabled give kafka more cpu and ram? Is it needed to manage the volume?